### DISCUSSION TITLE
May I get coords of text without paddings? FabricJS

### DISCUSSION THREAD

**Discussion by Retryno** [fabricjs/fabric.js#9182] [Q&A]
### Description

So I need to get coordinates of text without paddings(red square) but for example aCoords, lineCoords and etc get back coords with paddings (blue square) . 
![image](https://github.com/fabricjs/fabric.js/assets/36660457/f8418e1a-87a1-425a-9e79-f692e828571d)

So qs maybe someone has any idea how to do it? Maybe i can`t see methods which can help me?

### Current State

I am trying to get coords of text in FabricJS without paddings and make the same image on my server using [Imagick](https://www.php.net/manual/ru/class.imagick.php) by coords from fabricjs, but I don`t know how to get information about paddings, because every time when I try to scale the paddings become bigger in progressive.

With image I do not have any problem and looks perfect, because image don`t have aditional paddindgs like text...
Example:
Fabricjs
![image](https://github.com/fabricjs/fabric.js/assets/36660457/06cd71d5-9658-4260-a917-4219f07ace5a)

And Imagic by coords from fabric
![image](https://github.com/fabricjs/fabric.js/assets/36660457/a00dc100-5ca4-48b0-aa02-186330ec23db)

### Additional Context

Hope you can help me, because I stopped on this moment a long time.

Thanks.

**Comment by ShaMan123** [comment]
duplicate #9183
